src/components/Topbar/index.tsx

```diff
@@ -2,11 +2,19 @@ import styled from "styled-components";
 import Link from "../Link";
 import ROUTES from "@constants/routes";
 import APP from "@constants/app";
+import Image from "next/image";
+import Sitelogo from "../../../public/static/images/logo/warslive.png";
 
 const Navbar: React.FC = () => {
   return (
     <Container>
       <Section>
+        <Image
+          src={Sitelogo}
+          alt="logo of the site"
+          width="30px"
+          height="30px"
```

Comment:
a with and height is always in pixels so you dont need to add ``px`` in suffix of them